magnum_capi_helm/tests/test_driver.py

```diff
@@ -1189,6 +1189,9 @@ def _get_cluster_helm_standard_values(self):
             "machineSSHKeyName": None,
         }
 
+    @mock.patch.object(
+        driver.Driver, "_get_k8s_keystone_auth_enabled", return_value=False
```

Comment:
Nit: it would be good to have one test with return_value is True, but I don't want to block this one any longer!